### PR TITLE
Administrative units

### PR DESCRIPTION
**Pull request by cavemanpi**
Depends on https://github.com/duosecurity/duo_client_python/pull/62

### CODE REVIEW

File: duo_client/admin.py [hunk 1]
```diff
@@ -2092,17 +2157,31 @@ def update_integration(self,
...
+
+        (limit, offset) = self.normalize_paging_args(limit, offset)
+        if limit:
+            return self.json_api_call(
+                'GET',
+                '/admin/v1/admins',
+                {'limit': limit, 'offset': offset}
+            )
+
+        generator = self.json_paging_api_call('GET', '/admin/v1/admins', {})
```
Comment (xdesai) [inline duo_client/admin.py]: is there a reason this one doesn't follow the pattern of having `get_admins_iterator()`

Comment (xdesai) [reply duo_client/admin.py]: actually it looks like #66 also adds get_admins. We should probably pick one of these to have it and rip it out of the others

**Comment by cavemanpi** [comment]
My master branch which has my original bits got merged into this branch... Which isn't what I wanted. So it has a lot of stuff that doesn't make sense... I'll improve clarity shortly.

**Comment by cavemanpi** [comment]
Actually, nevermind. It turns out this branch was based off some initial work which made it into my local master but not duosecurity/master. I actually needed to merge my master back into this branch as it fell out of sync... 

@xdesai get_admins should look correct now.

File: duo_client/admin.py (hunk 1 above)
Comment (cavemanpi) [reply duo_client/admin.py]: Addressed in main comment thread.

File: tests/admin/test_administrative_units.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
+from .. import util
+from .base import TestAdmin
+
+
+class TestAdmins(TestAdmin):
```
Comment (xdesai) [inline tests/admin/test_administrative_units.py]: can you change this to TestAdminUnits? or something like that?

**Pull request merged**